<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>65</width>
        <height>26</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>85</width>
        <height>71</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="10" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_server">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>server ip:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pbGo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="11">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>670</width>
        <height>530</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>666</width>
         <height>500</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <attribute name="title">
        <string>views</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="FrameGL" name="glWidget">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QGraphicsView" name="gvDepth"/>
        </item>
        <item row="1" column="0">
         <widget class="QGraphicsView" name="gvBarre"/>
        </item>
        <item row="0" column="1">
         <widget class="QGraphicsView" name="gvVideo"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>data</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="8" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pbStop">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>88</width>
        <height>38</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="combo">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_k">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>select k</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>28</width>
        <height>25</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_ip">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>localhost</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMenu2">
   <property name="text">
    <string>opcion1_1</string>
   </property>
  </action>
  <action name="actionOpcion2_1">
   <property name="text">
    <string>opcion2_1</string>
   </property>
  </action>
  <action name="actionOpcion1_2">
   <property name="text">
    <string>opcion1_2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FrameGL</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">framegl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
